packages/models/src/shared/message.ts

```diff
@@ -0,0 +1,22 @@
+import type { MessageType } from './types';
+import { randomUUID } from 'crypto';
```

Comment:
This doesn't work in the browser, unless there are polyfills (which next.js may have added if you tested in that environment). We need to use the global `crypto.randomUUID()` and tell users this only works in node 18+.

Reply:
Is there canonical code that does import and catches a missing function to support both browsers and node?